frontend/src/players/players-list/players-list.tsx

```diff
@@ -80,24 +81,16 @@ const AddPlayerLink = styled(NavLink)``;
 
 type Props = {
   team: Team;
-  onFormSubmit: (team: Team) => void;
+  deletePlayer: (teamId: number, playerId: number) => void;
 };
 
-export const PlayersList = ({ team, onFormSubmit }: Props): JSX.Element => {
-  const [players, setPlayers] = useState<Player[]>([]);
-
-  const api = playersAPI();
+export const PlayersList = ({ team, deletePlayer }: Props): JSX.Element => {
+  const players = team.players ? team.players : [];
 
   const addPlayerUrl = `/teams/${team.id}/player/create`;
```

Comment:
I personally think, that it is more readable when you put URL in `NavLink` prop `to`. Your approach forced me to look up for variable `addPlayerUrl` which i missed at first glance.